utils/jmxfiles.py

```diff
@@ -0,0 +1,109 @@
+# std
+import logging
+import os
+import tempfile
+import time
+
+# 3rd party
+import yaml
+
+# datadog
+from config import _windows_commondata_path
+from util import yDumper
+from utils.pidfile import PidFile
+from utils.platform import Platform
+
+
+log = logging.getLogger(__name__)
+
+
+class JMXFiles(object):
```

Comment:
Under the hood, it works fine both ways anyway. I was under the influence of this video https://www.youtube.com/watch?v=o9pEzgHorH0 I watched a couple of weeks ago (same kind of ironic tone as the "Linux Sucks" conferences :) )